src/main/java/com/healthyfood/utils/CapitalizeTheFirstLetter.java

```diff
@@ -0,0 +1,7 @@
+package com.healthyfood.utils;
+
+public class CapitalizeTheFirstLetter {
```

Comment:
Do you plan to use this in more classes than one?

Reply:
I don't know yet, that's why I made a separate class.